pkgs/tools/misc/page/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "page";
+  version = "2.3.2";
+
+  src = fetchFromGitHub {
```

Comment:
Looking at the source repo, it seems that they also include a small program to generate shell completions. You could consider installing the completion files. `installShellFiles` is really nice for installing them:

https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/install-shell-files.sh

Reply:
thanks for the input. I added this in fecca1c